geomesa-gs-monitor-elasticsearch/src/main/scala/org/geomesa/gs/monitor/elastic/ExtendedRequestData.scala

```diff
@@ -79,6 +80,41 @@ class ExtendedRequestData(requestData: RequestData) extends RequestData(requestD
     distinguishedName
       .map(extractRdnValues(_, "OU"))
       .asNonEmptyJavaListOrNull
+
+  val resourceNames: java.util.List[String] =
+    requestData
+      .getResources
+      .asScala
+      .map(_.split(":")(1))
```

Comment:
it might be good to flatMap  and wrap this in a Try just in case the string isn't in the format we're expecting, so that the rest of the request would still get logged.

Reply:
I'm actually just going to remove this. After looking into the geoserver sources the methods are already there to extract what we need.